### DISCUSSION TITLE
Welcome to EthereumJS Discussions!

### DISCUSSION THREAD

**Discussion by ryanio** [ethereumjs/ethereumjs-monorepo#1424] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with members of our EthereumJS community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by acolytec3** [comment]
:wave: Hi all,I'm one of the newer devs on the EthereumJS team and am really excited on building out the `client` to add another execution client option to the Ethereum ecosystem.

**Comment by ghost** [comment]
hey all, I'm a solidity smart contract dev, and a casual open source contributor to projects such as geth and nimbus.
I hope to be able to contribute here too!

**Reply by ryanio** [reply]
👋 welcome!
